app/email_batching.go

```diff
@@ -201,15 +217,21 @@ func sendBatchedEmailNotification(userId string, notifications []*batchedNotific
 	}
 }
 
-func renderBatchedPost(template *utils.HTMLTemplate, post *model.Post, teamName string, displayNameFormat string, translateFunc i18n.TranslateFunc) string {
-	schan := Srv.Store.User().Get(post.UserId)
-	cchan := Srv.Store.Channel().Get(post.ChannelId, true)
```

Comment:
I moved the static access up to `sendBatchedEmailNotification(...)` so that I could more easily unit test `renderBatchedPost(...)` without having to modify the stores. Long-term, we should strive to get rid of all the static access code. It makes testing a real pain.